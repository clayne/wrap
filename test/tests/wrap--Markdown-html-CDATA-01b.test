wrap | /dev/null | -u | md-html-CDATA-01b.md | 0
