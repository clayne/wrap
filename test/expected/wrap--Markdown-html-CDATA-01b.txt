This is some text.  This is some more text.
<![CDATA[This is some CDATA.
This is some character data.
<html>
  This is some more character data.
  This data should not be wrapped.
</html>
This is yet more character data.
]]>
This is some text.  This is some more text.
